app/javascript/src/components/Dashboard/Contacts/NewContactPane.jsx

```diff
@@ -31,19 +30,17 @@ export default function NewContactPane({ showPane, setShowPane }) {
       >
         <Form>
           <Pane.Body>
-            <div>
+            <div className="space-y-3">
               <div className="flex space-x-2">
                 <Input
                   label="First Name"
                   name="firstName"
-                  className="mb-6"
                   required={true}
```

Comment:
For boolean values, if `true`, we can just pass `required`. 
No need to pass `required = {true}`.